src/Mono.Android/ILLink/ILLink.Substitutions.xml

```diff
@@ -4,7 +4,6 @@
       <method signature="System.Boolean get_VSAndroidDesignerIsEnabled()" body="stub" value="false" />
     </type>
     <type fullname="Java.Interop.TypeManager" feature="Android.Runtime.AndroidEnvironment.VSAndroidDesignerIsEnabled" featurevalue="false">
-      <method signature="System.Type TypeRegistrationFallback()" body="stub" value="null" />
```

Comment:
Instead of removing this line, we should instead *correct* it to match our actual method:

https://github.com/xamarin/xamarin-android/blob/0150bbbc36b3656b70bd22a3f8448e4cf23a8d65/src/Mono.Android/Java.Interop/TypeManager.cs#L234

This should presumably be:

```xml
<method signature="System.Type TypeRegistrationFallback(System.String)" body="stub" value="null" /
```

Reply:
It appears that ILLink also doesn't like `value="null"`:
```
resource ILLink.Substitutions.xml in Mono.Android, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065(7,8):
warning IL2010: Invalid value for 'Java.Interop.TypeManager.TypeRegistrationFallback(String)' stub
```

Reply:
See also: 
 
 * https://github.com/mono/linker/blob/54f4827313384e43bfd15c7ec6a45b6fa76090c3/src/linker/Linker.Steps/BodySubstitutionParser.cs#L90-L102
  * https://github.com/mono/linker/blob/54f4827313384e43bfd15c7ec6a45b6fa76090c3/src/linker/Linker.Steps/ProcessLinkerXmlBase.cs#L575-L578

I think we don't _want_ `value`, because `value` doesn't work the way we think it does.

Instead, let's remove the value attribute, and try:

```xml
<method signature="System.Type TypeRegistrationFallback(System.String)" body="stub" />
```

Reply:
> <method signature="System.Type TypeRegistrationFallback(System.String)" body="stub" />

Ok, yeah this is what I put for now.